doctrine/reverse_engineering.rst

```diff
@@ -85,6 +85,23 @@ run:
     // generates getter/setter methods
     $ php bin/console make:entity --regenerate App
 
+If you generated XML or YAML metadata you have also to change the ``config/packages/doctrine.yaml``
```

Comment:
Wouldn't this better fit in the `tip` block above where we show that you can run `doctrine:mapping:import` with xml or yml?

Reply:
I mean we can update it to look like this:

```rst
.. tip::

    It's also possible to generate the metadata files into XML or YAML:

    .. code-block:: terminal

        $ php bin/console doctrine:mapping:import 'App\Entity' xml --path=config/doctrine

    In this case, make sure to adapt your mapping configuration accordingly:

    .. code-block:: yaml

        # config/packages/doctrine.yaml
        # ...
```